calculator.py

```diff
@@ -0,0 +1,51 @@
+print("Calculator")
+result = 0
+run_again = True
+
+def calculate():
+    # global keyword allows us to modify the variable outside of the current scope
+    global result
+    if operation == "+":
+        result = num1 + num2
+    elif operation == "-":
+        result = num1 - num2
+    elif operation == "/":
+        if num2 == 0:
+            print("Not a number. Try again!")
+            return
+        else:
+            result = num1 / num2
+    elif operation == "x":
+        result = num1 * num2
+    print("Result:", result)
+
+ # this can be while True. Thought changing the value of run_again to false would
+ # cause the session to end   
+while run_again:
+    operations = ["+", "-", "/", "x"]
```

Comment:
**praise**: good use of list. storing all the operations handled by the program.